tests/e2e/pageobjects/dashboard/Workspaces.ts

```diff
@@ -192,4 +211,7 @@ export class Workspaces {
         return By.xpath(`${this.getWorkspaceListItemLocator(workspaceName)}//td[@data-key=5]//a[text()='Open']`);
     }
 
+    private getWorkspaceItemNameTableSectionLocator() {
```

Comment:
Please consider changing this to a constant

Reply:
thnx, done